website/Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: default
+default: generated
+
+CRD_REF_DOCS?=github.com/elastic/crd-ref-docs@v0.1.0
```

Comment:
We should have a variable for the version and have a mechanism to automatically bump it (that part can be done later).